site/content/contribute/desktop/developer-setup.md

```diff
@@ -12,12 +12,46 @@ subsection: "Desktop App"
 - NPM
 - Git
 
+### Windows
+Recommended way to install prerequisites on windows is to install chocolatey package manager.
+once installed run from an administrator powershell console
+```
+choco install nvm
+```
+```
+choco install git
+```
+from a non privileged powershell console:
+```
+nvm install 10.18.1
+```
+
 ## Installing dependencies
 After installation, dependencies of `src/` directory are also installed.
 
+### OSX & Linux
 ```
 $ npm install
 ```
+### Windows
+on an administrator powershell run
+
+```
+> .\scripts\Makefile.ps1 Install-Deps
+```
+
+#### Known problems
```

Comment:
```suggestion
#### Known Limitations
```

Reply:
isn't this a usual section for manuals? anyway, this is not a limitation but issues I encountered while running the above commands. With this section we aim to provide workarounds for people in case they run into the same problem.

Reply:
That makes a lot of sense, so I've changed it to "Known Problems". :)